src/libecalc/dto/components.py

```diff
@@ -190,8 +190,17 @@ class PumpSystemOperationalSetting(EcalcBaseModel):
     outlet_pressures: List[Expression]
 
 
+class SystemCrossoverStream(EcalcBaseModel):
+    class Config:
+        allow_population_by_field_name = True
+
+    stream_name: Optional[str] = Field(None)
+    from_component_id: str
```

Comment:
I would want to have a special syntax on references, to make it clear that it is a reference, such as prefix with $ (which is basically a standard?!). Then it would make sense to define our own type for the places where we require "references"...

Reply:
Would a reference meaning "insert reference here" (i.e. yaml include-ish) be different from this reference where we mean "from the component with this name" (but not insert the yaml)